uktrade/data-hub-leeloo
# Rollback removal of assign pm stage

### Description of change
This removes the work done to remove the `Assign PM` from the investment flow. The work was released behind a feature flag which was never activated. 
The future plan is to add triaging in to the investment flow so these changes will no longer be required.

### To test
Test everything works the same
- Create an investment project
- On saving the prospect you should then be told what data is needed to move to `Assign PM`
- Fill in the required data
- Move project to `Assign PM`
- Fill in `Project manager` and `Project Assurance Adviser` and move to `Active`


### Checklist
